test/common/tests.go

```diff
@@ -95,9 +96,7 @@ var (
 		{"F08 - Verify Replicas Scale correctly in User SSO", TestReplicasInUserSSO},
 	}
 
-	FAILURE_TESTS = []TestCase{
-		/*FLAKY*/ {"C03 - Verify that alerting mechanism works", TestIntegreatlyAlertsMechanism},
-	}
+	FAILURE_TESTS = []TestCase{}
```

Comment:
Why was this moved out of the failure tests. This is not a happy path test as it test functionally of a install that is not in a healthy state.

Reply:
Reason for this is that I had a different understanding of what "FAILURE_TESTS" are. Thanks for explaining what it is!

Reply:
I've also updated the prow e2e to run the FAILURE_TESTS as it seems that we were not executing this test on a PR submission.